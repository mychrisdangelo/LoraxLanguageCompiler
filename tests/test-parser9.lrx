/*
 * Author: Chris D'Angelo
 * Testing valid parseable file
 */

int main()
{
t = 1[2[1], 3];
return 0;
}