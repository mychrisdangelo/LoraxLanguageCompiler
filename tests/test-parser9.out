int main()
{
t = 1[2[1], 3];
return 0;
}